tests/infra_test.go

```diff
@@ -666,120 +667,143 @@ var _ = Describe("Infrastructure", func() {
 				Transport: tr,
 			}
 
-			errors := make(chan error)
-			for ix := 0; ix < concurrency; ix++ {
-				go func(ix int) {
-					req, _ := http.NewRequest("GET", metricsURL, nil)
-					resp, err := client.Do(req)
-					if err != nil {
-						fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
-					} else {
-						resp.Body.Close()
-					}
-					errors <- err
-				}(ix)
-			}
+			for _, ip := range metricsURLs {
+				if strings.Index(ip, "[") != -1 {
+					By("Skipping Scraping the Prometheus endpoint (IPv6), see https://github.com/kubevirt/kubevirt/issues/4145")
+					continue
```

Comment:
i will try to use Skip, just not sure it wont skip the whole test in case one is skipped,
I remember vaguely i saw this behavior